### DISCUSSION TITLE
Is there a way to control sources?

### DISCUSSION THREAD

**Discussion by rubenesque-code** [cloudinary-community/next-cloudinary#139] [Q&A] [answered]
Hi there. Thanks for this package.

I'm wondering if there's a way to limit the sources provided by `CldUploadWidget`? I only really need local images as a source.

Cheers.

**Comment by colbyfayock** [comment, marked as answer]
Yes! Just tested this out, and the `options` prop works perfectly to take this parameter

```
options={{
  sources: ['local']
}}
```

I just added an example to show how: https://next-cloudinary.spacejelly.dev/components/clduploadwidget/examples#custom-sources


More options for the `sources` parameter and other options:

https://cloudinary.com/documentation/upload_widget#the_sources_parameter

**Reply by rubenesque-code** [reply]
Thanks! I got a bit stuck because it's not typed whilst another property (`uploadSignature`) is.

**Reply by colbyfayock** [reply]
creating an issue to add this :)
